### PR TITLE
No change wellcomelibrary.org passthrough

### PR DESCRIPTION
**Pull request by kenoir**
Simplest no-change update to redirect requests via CloudFront.

This change removes any extras and establishes a baseline no-change switcheroo for requests coming via wellcomelibrary.org that introduces CloudFront as an intermediary.

Note: This change is applied.

### CODE REVIEW

File: cloudfront/wellcomelibrary.org/terraform/dns.tf [hunk 1]
```diff
@@ -1,9 +1,45 @@
...
+  type    = "A"
+
+  weighted_routing_policy {
+    weight = 50
+  }
+
+  alias {
+    name                   = "d2cksn8op995w4.cloudfront.net"
+    evaluate_target_health = false
+    zone_id                = "Z2FDTNDATAQYW2"
```
Comment (alexwlchan) [inline cloudfront/wellcomelibrary.org/terraform/dns.tf:29]: Where does this Zone ID come from?

File: cloudfront/wellcomelibrary.org/terraform/dns.tf [hunk 2]
```diff
@@ -1,9 +1,45 @@
...
+  zone_id = data.aws_route53_zone.zone.id
+  name    = "wellcomelibrary.org"
+  type    = "A"
+
+  weighted_routing_policy {
+    weight = 50
+  }
+
+  alias {
+    name                   = "d2cksn8op995w4.cloudfront.net"
```
Comment (alexwlchan) [inline cloudfront/wellcomelibrary.org/terraform/dns.tf:27]: Where does this CloudFront name come from?

File: cloudfront/wellcomelibrary.org/edge-lambda/src/redirect.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import {CloudFrontResultResponse} from "aws-lambda/common/cloudfront";
+
+export async function redirect(uri: string){
+    return {
+        status: '301',
```
Comment (alexwlchan) [inline cloudfront/wellcomelibrary.org/edge-lambda/src/redirect.ts:5]: Two minor thoughts:

* I wonder if we should stick to 302 until we've got the redirection stuff sorted, so clients don't end up caching the redirects?
* Why is this function async?

**Review by alexwlchan — APPROVED** [review]

File: cloudfront/wellcomelibrary.org/edge-lambda/src/redirect.ts (hunk 3 above)
Comment (kenoir) [reply cloudfront/wellcomelibrary.org/edge-lambda/src/redirect.ts:5]: - Good idea on 302.
- The function need not be async.

File: cloudfront/wellcomelibrary.org/terraform/dns.tf (hunk 1 above)
Comment (kenoir) [reply cloudfront/wellcomelibrary.org/terraform/dns.tf:29]: Good catch - I copy/pasted this to satisfy a terraform import and didn't update it. I can just pull this from the prod CloudFront distro module 👍

File: cloudfront/wellcomelibrary.org/terraform/dns.tf (hunk 2 above)
Comment (kenoir) [reply cloudfront/wellcomelibrary.org/terraform/dns.tf:27]: Same issue as with zone_id.

File: cloudfront/wellcomelibrary.org/terraform/dns.tf (hunk 1 above)
Comment (kenoir) [reply cloudfront/wellcomelibrary.org/terraform/dns.tf:29]: Oh, this is an interesting one actually - this value is the same for all cloudfront distros (added a comment with link to docs).

**Pull request merged**